### PR TITLE
PRO - 1282 : Remove tooltips from wallet

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1282/pillar-wallet-remove-tooltips-from-wallet

## Description

- Removed some tooltips in home screen
  1. Switch wallet
  2. Add cash
  3. Sign in defi

## How Has This Been Tested?

- Tested on Simulator 12 

## Screenshots (if appropriate):
https://user-images.githubusercontent.com/82652040/230892167-164359d6-0b5c-4465-8c4e-af27e56deed2.mp4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by IAmKio — APPROVED** [review]
LGTM!

File: src/screens/Home/Home.js [hunk 1]
```diff
@@ -250,17 +238,8 @@ function Home() {
...
-              wrapperStyle={{ zIndex: 9999, top: -10, position: 'relative' }}
-            />
-          )}
-
           <Tooltip
-            isVisible={!switchAccountTooltipDismissed && isKeyBasedAccount && !showAccountSwitchTooltip}
+            isVisible={!switchAccountTooltipDismissed && isKeyBasedAccount}
             body={t('tooltip.switch_account')}
             wrapperStyle={{ zIndex: 9999, top: -10, position: 'relative' }}
           />
```
Comment (Yash-Chaubal) [inline src/screens/Home/Home.js:245]: Could this also be removed?

**Review by Yash-Chaubal — APPROVED** [review]
LGTM! Just left a question

File: src/screens/Home/Home.js (hunk 1 above)
Comment (lalitkumarrock) [reply src/screens/Home/Home.js:245]: This is not mentioned in ticket. I think this is part of wallet connect. toast message is "Switched to your Key Wallet for the Etherspot Dashboard"

**Review by MayukhSamadder2000 — APPROVED** [review]
LGTM!

**Pull request merged**